### PR TITLE
DTGB-509: Basic filter component and -page.

### PR DESCRIPTION
**Pull request by delrueba**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: styleguide/components/31-molecules/fields/checkbox-with-filter/checkbox-with-filter.twig [hunk 1]
```diff
@@ -5,31 +5,38 @@
       <span class="label-optional">({{ label_optional }})</span>
     {% endif %}
   </legend>
+  {% if field_description %}
   <div class="form-columns">
```
Comment (meirege) [inline styleguide/components/31-molecules/fields/checkbox-with-filter/checkbox-with-filter.twig]: Indentation

File: styleguide/components/31-molecules/fields/checkbox-with-filter/checkbox-with-filter.twig [hunk 2]
```diff
@@ -5,31 +5,38 @@
...
+            {% endif %}
+          </h3>
         </header>
         <div class="form-item">
           <label for="checkboxes__filter_id_{{ modifier }}">Filter the list below</label>
           <input type="search" id="checkboxes__filter_id_{{ modifier }}"
                  class="checkbox-filter__filter">
-          <strong aria-live="polite" class="checkbox-filter__result-wrapper">There are <span class="checkbox-filter__result">#</span> results</strong>
-        </div>
+          <div class="checkbox-filter__selected">
```
Comment (meirege) [inline styleguide/components/31-molecules/fields/checkbox-with-filter/checkbox-with-filter.twig]: Remove whitespace.

File: styleguide/components/31-molecules/fields/checkbox-with-filter/checkbox-with-filter.twig [hunk 3]
```diff
@@ -62,6 +72,8 @@
         "modifier": "success"
         } %}
       {% endif %}
-    </div>
+  {% if field_description %}
+      </div>
   </div>
+  {% endif %}
```
Comment (meirege) [inline styleguide/components/31-molecules/fields/checkbox-with-filter/checkbox-with-filter.twig]: Indentation.

**Review by meirege — CHANGES_REQUESTED** [review]

**Review by meirege — APPROVED** [review]

File: styleguide/components/11-base/base/base.js [hunk 4]
```diff
@@ -9,7 +9,6 @@ window.gent_styleguide = (function () { // eslint-disable-line no-unused-vars
    * @constructor
    */
   function TabTrap(container) {
-    var focusPosition = -1;
     var focusables = getFocusables(container);
```
Comment (Jeroen005) [inline styleguide/components/11-base/base/base.js]: horen deze vars geen const te zijn?

**Review by Jeroen005 — CHANGES_REQUESTED** [review]

File: styleguide/components/11-base/base/base.js (hunk 4 above)
Comment (delrueba) [reply styleguide/components/11-base/base/base.js]: This file predates our shift to ecmascript.

**Review by Jeroen005 — APPROVED** [review]

**Pull request merged**